2024-12-03 18:44:19,602 - DEBUG - Using proactor: IocpProactor
2024-12-03 18:44:19,602 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 18:44:19,868 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 18:44:19,868 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 18:51:07,741 - DEBUG - Using proactor: IocpProactor
2024-12-03 18:51:07,741 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 18:51:07,819 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 18:51:07,819 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 18:53:51,994 - DEBUG - Using proactor: IocpProactor
2024-12-03 18:53:51,994 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 18:53:52,057 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 18:53:52,057 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 18:57:56,306 - DEBUG - Using proactor: IocpProactor
2024-12-03 18:57:56,322 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 18:57:56,400 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 18:57:56,400 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 18:58:02,914 - DEBUG - Using proactor: IocpProactor
2024-12-03 18:58:02,914 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 18:58:02,964 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 18:58:02,980 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 18:59:38,029 - DEBUG - Using proactor: IocpProactor
2024-12-03 18:59:38,029 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 18:59:38,107 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 18:59:38,107 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 19:02:58,604 - DEBUG - Using proactor: IocpProactor
2024-12-03 19:02:58,619 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 19:02:58,682 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 19:02:58,682 - WARNING - Conectividade com https://web.dio.me/articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest falhou. Status code: 404
2024-12-03 19:02:58,682 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 19:05:25,548 - DEBUG - Using proactor: IocpProactor
2024-12-03 19:05:25,548 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 19:05:25,626 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 19:05:25,626 - WARNING - Conectividade com https://web.dio.me/articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest falhou. Status code: 404
2024-12-03 19:05:25,626 - ERROR - Falha na verificação de conectividade com a internet.
2024-12-03 19:07:23,049 - DEBUG - Using proactor: IocpProactor
2024-12-03 19:07:23,049 - DEBUG - Starting new HTTPS connection (1): web.dio.me:443
2024-12-03 19:07:23,315 - DEBUG - https://web.dio.me:443 "GET /articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest HTTP/11" 404 7297
2024-12-03 19:07:23,315 - WARNING - Conectividade com https://web.dio.me/articles/programa-em-python-para-automacao-de-pesquisas-no-edge?back=%2Fhome&page=1&order=oldest falhou. Status code: 404
2024-12-03 19:07:23,315 - ERROR - Falha na verificação de conectividade com a internet.
